modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java

```diff
@@ -174,15 +173,6 @@ public Map<String, PropertyDefinition> getExtendedPropertyDefinitions(
 				continue;
 			}
 
-			PropertyValidator propertyValidator = null;
-
-			if (FeatureFlagManagerUtil.isEnabled("LPS-153117")) {
-				propertyValidator = new DefaultPropertyValidator();
-			}
-			else {
-				propertyValidator = new UnsupportedOperationPropertyValidator();
```

Comment:
I think the private class `UnsupportedOperationPropertyValidator` could be removed as I think we are not using it anymore

Reply:
:facepalm: that's true, thanks @ccorreagg!!